gssapi/tests/test_raw.py

```diff
@@ -474,17 +421,17 @@ def test_store_cred_acquire_cred(self):
                                                 acceptor_creds=server_creds)
 
         deleg_creds = server_ctx_resp.delegated_creds
-        deleg_creds.shouldnt_be_none()
+        self.assertIsNotNone(deleg_creds)
+
         store_res = gb.store_cred(deleg_creds, usage='initiate',
                                   set_default=True, overwrite=True)
-
-        store_res.shouldnt_be_none()
-        store_res.usage.should_be('initiate')
-        store_res.mechs.should_include(gb.MechType.kerberos)
+        self.assertIsNotNone(store_res)
```

Comment:
I guess you could go through and look at these too -- `None` would never equal `"initiate"`. My 2c.

Reply:
Careful - the next line is checking `store_res.usage`, not `store_res`.

Reply:
Ah ok. Still, `None` is its own type in Python so `store_res.usage` would actually fail prior to checking `initate` value. This works though.

Reply:
Yes, but it fails with an exception, not an assertion failure.  I find the assertion failures a lot easier to work with when I can get them :)

Reply:
Yeah that's one thing I didn't like about this test suite -- it ran and gave me > 40 failures (when my hostname was wrong) versus saying somthing like "I have encountered an error, here it is" and spit it out for me while I'm running it. Even if it continues running (JSS does), it should still print more than "ERROR" and leave me in suspense about what the error is :-)

Reply:
Would you be able to file a bug for that?